src/components/MarkdownStyler.tsx

```diff
@@ -15,7 +16,7 @@ import rehypeRaw from 'rehype-raw';
  * @params allowHtml Whether to parse raw html tags in content. DANGEROUS!
  */
 export default function MarkdownStyler({ content, allowHtml }: {
-  content: string,
+  content: string | ImageParams,
```

Comment:
why do you need to change this file? we should not need to change this file for this feature. It doesn't make sense for MarkdownStyler to take ImageParams as content type.

Instead, use `context as string` at where MarkdownStyler is called.